### DISCUSSION TITLE
How to show only list of files (without listed matching lines)?

### DISCUSSION THREAD

**Discussion by karolzlot** [BurntSushi/ripgrep#2206] [Q&A] [answered]
I would like to use ripgrep like: `rg "some phrase"`

The problem is it it prints filepaths and also lines from those files:
```
abc_dir\workspace.json
2:  "abc": "some phrase abc"

cde_dir\workspace.json
2:  "123": "some phrase def"
67: hhhh some phrase hjnbvtr
```

But I would like to get only list of files like this:
```
abc_dir\workspace.json
cde_dir\workspace.json
```
How can I achieve it with ripgrep?

**Comment by BurntSushi** [comment, marked as answer]
The same way you'd do it with plain on `grep`: via the `-l/--files-with-matches` flag. So `rg -l "some phrase"`.
